scripts/run_integration_tests_aarch64.sh

```diff
@@ -217,19 +248,7 @@ update_workloads() {
     guestunmount "$FOCAL_OS_RAW_IMAGE_UPDATE_KERNEL_ROOT_DIR"
 
     # Build virtiofsd
-    VIRTIOFSD="$WORKLOADS_DIR/virtiofsd"
-    VIRTIOFSD_DIR="virtiofsd_build"
-    if [ ! -f "$VIRTIOFSD" ]; then
-        pushd $WORKLOADS_DIR
-        git clone "https://gitlab.com/virtio-fs/virtiofsd.git" $VIRTIOFSD_DIR
-        pushd $VIRTIOFSD_DIR
-        git checkout v1.1.0
-        time cargo build --release
-        cp target/release/virtiofsd $VIRTIOFSD || exit 1
-        popd
-        rm -rf $VIRTIOFSD_DIR
-        popd
-    fi
+    build_virtiofsd
```

Comment:
I thought this was already removed. We are only testing Rust based virtiofsd.

Reply:
Yes, the C version virtiofsd was removed in the dockerfile. But I think the rust version is built in both x86 and arm integration test scripts

Reply:
Whoops. Apologires I forgot that the Rust version was renamed. You're spot on. Does this code correctly rebuild if the commit hash is changed?

Reply:
Yes, if the hash is changed, the checkout_repo will delete the source dir and therefore lead to a new build of the binary.